e2e/README.md

```diff
@@ -1,42 +1,39 @@
-## Teleport E2E Tests
+# E2E Testing with Playwright
 
-This directory contains end-to-end tests for Teleport. These tests are
-designed to be run against a live cluster. They are written in TS and use
-[Playwright](https://playwright.dev/) to interact with the browser.
-Docker compose is used to spin up a cluster for testing and to run the tests.
+This directory contains the configuration and tests for end-to-end testing against a real Teleport instance using Playwright.
 
-### Running the tests
-```bash
-# Make all removes the existing docker volumes to ensure a clean state
-# and rebuild the containers
-make all 
-```
-or
+E2E tests should be run by a corresponding Go test in `web-e2e_test.go` and run
```

Comment:
I'd mention here the role of `START_URL`. Each test starts from setting up an account, so `START_URL` must contain an invite token (which is supplied by the Go tests).